src/components/binding/bindApplicationForm.html

```diff
@@ -22,7 +22,7 @@
             You can create the binding later from your project.
           </span>
         </div>
-        <div ng-repeat="serviceInstance in ctrl.serviceInstances">
+        <div ng-if="ctrl.bindableServiceInstances.length" ng-repeat="serviceInstance in ctrl.bindableServiceInstances">
```

Comment:
Do we need the `ng-if`? The `ng-repeat` should have no output if there are no bindable instances